docs/BackupAndRestore.md

```diff
@@ -103,6 +103,42 @@ Use `nuobackup --type report-latest --db-name <db> --group <backup group>` to sh
 It is possible to disable backup jobs creation by setting `hotcopy.enableBackups` to `false`.
```

Comment:
While I've spotting it, should be "hotCopy.enableBackups"

Reply:
Good catch!